docs/index.md

```diff
@@ -19,6 +19,7 @@ portable applications for AMD and NVIDIA GPUs from single source code.
 
 * {doc}`./understand/programming_model`
 * {doc}`./understand/programming_model_reference`
+* {doc}`./reference/virtual_rocr`
```

Comment:
If you add it to Reference, please remove from "Conceptual" index card.  We don't want 2 entries on the index page.

Reply:
Nice catch, I pushed the fix.

Reply:
Looks better.  Thx for the fix.